docusaurus.config.js

```diff
@@ -28,6 +31,23 @@ const onlyIncludeVersions = isDev
 const currentVersion = getCurrentVersion();
 const includeBlog = process.env.PANTSBUILD_ORG_INCLUDE_BLOG === "1" || !isDev;
 
+const formatCopyright = () => {
+  const year = new Date().getFullYear();
```

Comment:
I know using the current year is common practice, but it's not actually correct. The copyright year should reflect the year of first publication, not the current year (since that conveys no information relevant to the content for which common law copyright is being claimed). So to be meaningful year should be the year of first publication, or the last time the file was substantially modified (if that can be measured). 

An easier possibility is to omit the year entirely...

Reply:
Removed!